ksp_plugin_adapter/localization/fr-fr.cfg

```diff
@@ -0,0 +1,241 @@
+    #Principia_OrbitAnalyser_Elements_LongitudeOfAscendingNode = Longitude du nœud ascendant
+    #Principia_OrbitAnalyser_Elements_NodalPrecession = Précession nodale
+    // These are celestial strings parameterized on the body.
+    #Principia_OrbitAnalyser_Elements_Periapsis = périastre
+    #Principia_OrbitAnalyser_Elements_Apoapsis = apoastre
+    // In the following three strings, <<1>> is the Periapsis or Apoapsis string
+    // above.
+    #Principia_OrbitAnalyser_Elements_ArgumentOfPeriapsis = Argument du <<1>>
+    #Principia_OrbitAnalyser_Elements_MeanPeriapsisAltitude = Altitude moyenne du <<1>>
+    #Principia_OrbitAnalyser_Elements_MeanApoapsisAltitude = Altitude moyenne de l’<<1>>
+    #Principia_OrbitAnalyser_GroundTrack = Trace au sol
+    #Principia_OrbitAnalyser_Recurrence_CapderouTriple = Phasage: <<1>>
+    #Principia_OrbitAnalyser_Recurrence_AutoDetect = Détecter automatiquement
+    #Principia_OrbitAnalyser_Recurrence_Cycle = Cycle
+    #Principia_OrbitAnalyser_Recurrence_Cycle_RevolutionsEquals = révolutions \u003D
+    #Principia_OrbitAnalyser_Recurrence_Cycle_Days = jours
+    #Principia_OrbitAnalyser_Recurrence_Subcycle = Sous-cycle
+    #Principia_OrbitAnalyser_Recurrence_SubcycleLengthInDays = <<1>> jours
+    #Principia_OrbitAnalyser_Recurrence_EquatorialShift = Décalage équatorial
+    #Principia_OrbitAnalyser_Recurrence_GridInterval = Intervalle de grille
+    #Principia_OrbitAnalyser_GroundTrack_LongitudesOfEquatorialCrossings_Prefix = Longitudes des passages a l’équateur de la rév. nᵒ
+    #Principia_OrbitAnalyser_GroundTrack_LongitudesOfEquatorialCrossings_Suffix =
+    #Principia_OrbitAnalyser_GroundTrack_AscendingPass = Passe ascendante
+    #Principia_OrbitAnalyser_GroundTrack_DescendingPass = Passe descendante
+    #Principia_OrbitAnalyser_MissionDuration = Durée
+    #Principia_CurrentOrbitAnalyser_ToggleButton = Analyse d’orbite...
+    #Principia_CurrentOrbitAnalyser_ToggleButtonWithDescription = Analyse : <<1>>...  // <<1>> orbit_description
+    #Principia_CurrentOrbitAnalyser_Analysing = Analyse de l’orbite de <<1>>...  // <<1>> vessel name
+    #Principia_PlannedOrbitAnalyser_ToggleButton = Analyse de la trajectoire finale...
+    #Principia_PlannedOrbitAnalyser_ToggleButtonWithDescription = Analyse de l’orbite finale : <<1>>...  // <<1>> orbit_description
+    #Principia_PlannedOrbitAnalyser_Analysing = Analyse de l’orbite finale planifiée pour <<1>>...  // <<1>> predicted_vessel.vesselName
+
+    // FlightPlanner
+    #Principia_FlightPlan_PlanLength = Longueur
+    #Principia_FlightPlan_ToggleButton = Plan de vol...
+    #Principia_FlightPlan_Title = Plan de vol
+    #Principia_FlightPlan_Create = Créer un plan de vol
+    #Principia_FlightPlan_MaxSteps = Max. pas par segment :
+    #Principia_FlightPlan_TotalΔv = Δv total : <<1>> m/s  // <<1>>: Δv.ToString("0.000").
+    #Principia_FlightPlan_Delete = Supprimer le plan de vol
+    #Principia_FlightPlan_Rebase = Rebaser
```

Comment:
Refonder?

Reply:
Rebaser appears to be the translation for the source control sense of _rebase_ which we used for the original.
Alternatively we could be more explicit, there is room (_Recontruire à partir de l’état actuel_ maybe?).

Reply:
Just "reconstruire" is good I think.